### DISCUSSION TITLE
希望增加音标以及词性和用法

### DISCUSSION THREAD

**Discussion by wjmboss** [YiiGuxing/TranslationPlugin#5841] [Ideas]
希望增加音标以及词性和用法,音标对于直接看怎么发音太有用了,词性和用法也非常重要,要是能把词缀再列出来那就是完美

**Comment by YiiGuxing** [comment]
这个取决于翻译服务供应商，它提供就有，不提供那我也没有办法啊，谷歌翻译就提供得有，但国内无法访问。

**Reply by wjmboss** [reply]
Edge翻译有提供吗?

**Reply by YiiGuxing** [reply]
没有，你看它的网页版，也是没有提供的，中文拼音它倒是提供了，但是对我们来说没用啊！

**Comment by khdmls** [comment]
哈哈，音标的需求+1，词性和用法感觉不是特别强烈，毕竟不是学习单词的软件
